src/main/scala/beam/sim/RideHailFleetInitializer.scala

```diff
@@ -143,7 +143,15 @@ class RideHailFleetInitializer extends LazyLogging {
         )
       val fileHeader = classOf[RideHailAgentInputData].getDeclaredFields.map(_.getName).mkString(", ")
       val data = fleetData map { f =>
-        f.productIterator.map(f => if (f == None) "" else f) mkString ", "
+        f.productIterator.map(
```

Comment:
This can use `map`s partial function overload:

```
f.productIterator.map{
  case Some(x:String) => x
  case x: String       => x
  case None            => ""
  case _               => f
}
```